### PR TITLE
Block obfuscated scripts from running

### PR DESCRIPTION
**Pull request by gerbsec**

### CODE REVIEW

File: tests/53ff0bda-537e-4493-90cf-220145d9569a/53ff0bda-537e-4493-90cf-220145d9569a.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import (
+	"fmt"
+	"github.com/preludeorg/test/endpoint"
+	"os"
+	"runtime"
+)
+
+func test() {
+	if runtime.GOOS == "windows" {
```
Comment (bfuzzy1) [inline tests/53ff0bda-537e-4493-90cf-220145d9569a/53ff0bda-537e-4493-90cf-220145d9569a.go:16]: You don't need runtime the run function in endpoint should handle that if I'm not mistaken.

Comment (privateducky) [reply tests/53ff0bda-537e-4493-90cf-220145d9569a/53ff0bda-537e-4493-90cf-220145d9569a.go:16]: correct. 
note that this is using powershell vs cmd, in the Run function. we should adjust to cmd syntax.

Comment (gerbsec) [reply tests/53ff0bda-537e-4493-90cf-220145d9569a/53ff0bda-537e-4493-90cf-220145d9569a.go:16]: Yeah I saw that, thing is I have 2 seperate b64 payloads cause windows is slightly different and 2 different commands that need to be run. The endpoint module only handles the binary that is running the command.

Comment (privateducky) [reply tests/53ff0bda-537e-4493-90cf-220145d9569a/53ff0bda-537e-4493-90cf-220145d9569a.go:16]: hmm, confused as to why we'd have separate base64 strings, based on platform

Comment (gerbsec) [reply tests/53ff0bda-537e-4493-90cf-220145d9569a/53ff0bda-537e-4493-90cf-220145d9569a.go:16]: https://nvd.codes/post/correctly-encode-powershell-payloads-for-exploits/ @privateducky

Comment (privateducky) [reply tests/53ff0bda-537e-4493-90cf-220145d9569a/53ff0bda-537e-4493-90cf-220145d9569a.go:16]: thanks - this helps

**Comment by privateducky** [comment]
this tested on all the things?

**Comment by bfuzzy1** [comment]
> this tested on all the things?

All tests are passing.

Tested:

```
Darwin arm64 test passed
Windows 11 arm64 test passed
Fedora 36 arm64 test passed
Debian 11.3 arm64 test passed
Ubuntu 22.04 ARM64 test passed
Windows 11 x64 test passed
Windows 10 x64 test passed
Windows Server 2016 test passed
Windows Server 2019 test passed
Ubuntu 22.04 amd64 test passed
Ubuntu 22.10 amd64 test passed
```

**Pull request merged**